core/templates/dev/head/css/oppia.css

```diff
@@ -682,6 +682,7 @@ textarea {
 .oppia-sidebar-menu-open .oppia-sidebar-menu {
   box-shadow: 1px 0 3px rgba(0,0,0,0.12), 1px 0 2px rgba(0,0,0,0.24);
   -webkit-transform: translate3d(0, 0, 0);
+  overflow-y: scroll;
```

Comment:
Done but as point of discussion if are using alphabetical to arrange css properties, shouldn't -webkit-transform come at starting.